src/pages/Xvs/Table/index.tsx

```diff
@@ -1,9 +1,103 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
-import { Paper } from '@mui/material';
+import React, { useContext, useMemo } from 'react';
+import { useHistory } from 'react-router-dom';
+import BigNumber from 'bignumber.js';
+import { Typography } from '@mui/material';
+import { useUserMarketInfo } from 'clients/api';
+import { Token, Table, ITableProps } from 'components';
+import { AuthContext } from 'context/AuthContext';
+import { useTranslation } from 'translation';
+import { Asset, TokenId } from 'types';
+import { formatToReadablePercentage, formatCoinsToReadableValue } from 'utilities/common';
+import { useStyles } from '../styles';
 
-const TableUi: React.FC = () => <Paper />;
+interface IXvsTableProps {
+  assets: Asset[];
+}
 
-const Table: React.FC = () => <TableUi />;
+const XvsTableUi: React.FC<IXvsTableProps> = ({ assets }) => {
+  const history = useHistory();
+  const { t } = useTranslation();
+  const styles = useStyles();
 
-export default Table;
+  const columns = useMemo(
+    () => [
+      { key: 'asset', label: t('xvs.columns.asset'), orderable: false },
+      { key: 'xvsPerDay', label: t('xvs.columns.xvsPerDay'), orderable: true },
+      { key: 'supplyXvsApy', label: t('xvs.columns.supplyXvsApy'), orderable: true },
+      { key: 'borrowXvsApy', label: t('xvs.columns.borrowXvsApy'), orderable: true },
+    ],
+    [],
+  );
+
+  const rowOnClick = (e: React.MouseEvent<HTMLElement>, row: ITableProps['data'][number]) => {
+    history.push(`/market/${row[0].value}`);
+  };
+  // Format assets to rows
+  const rows: ITableProps['data'] = assets.map(asset => [
+    {
+      key: 'asset',
+      render: () => <Token symbol={asset.symbol as TokenId} />,
+      value: asset.id,
+    },
+    {
+      key: 'xvsPerDay',
+      render: () => (
+        <Typography variant="small1" css={styles.whiteText}>
+          {formatCoinsToReadableValue({
+            value: asset.xvsPerDay.dp(2, 1),
```

Comment:
`formatCoinsToReadableValue` already takes care of the decimal places.
I guess you did that to follow the designs, but they haven't been updated following some of the updates we've made regarding value formatting so I'd rather stay consistent with the rest of the app and use the `shorthand` option here.